src/App.jsx

```diff
@@ -14,70 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood === 'clear'
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              onClick={() => {
+                setSelectedGood('clear');
```

Comment:
and set empty string here
```suggestion
                setSelectedGood('');
```